src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -955,6 +946,23 @@ private static boolean startsAfter(CompactionSSTable a, CompactionSSTable b)
         return a.getFirst().compareTo(b.getLast()) > 0;
     }
 
+    private Collection<AbstractCompactionTask> getZombieRepairFinalizationTasks(Collection<? extends CompactionSSTable> sstables)
```

Comment:
Could we move this into `LocalSessions`? It already contains very similar methods (`cleanup`, `releaseRepairData` etc.)

Reply:
Yep sure. Having a method taking sstables to cleanup will be useful.